Marlin/ultralcd.cpp

```diff
@@ -3334,14 +3334,11 @@ void lcd_init() {
       SET_INPUT_PULLUP(BTN_ENC);
     #endif
 
-	  #if ENABLED(ADC_KEYPAD)
-	  #else
-    #if ENABLED(REPRAPWORLD_KEYPAD)
+    #if ENABLED(REPRAPWORLD_KEYPAD) && DISABLED(ADC_KEYPAD)
```

Comment:
Sure.. I think the other way was used to not touch original lines for merges/rebases